bundles/org.openhab.core.model.script/bnd.bnd

```diff
@@ -64,3 +64,4 @@ Require-Bundle: org.antlr.runtime,\
  org.eclipse.xtext.xbase;visibility:=reexport,\
  org.eclipse.xtext;visibility:=reexport,\
  org.objectweb.asm;resolution:=optional
+ -includeresource: -${resources}
```

Comment:
This line didn't add ESH-INF to the JAR on a Maven build for me.

What did work was:

```suggestion
-includeresource: resources
```

Reply:
Not sure why it worked for me, but reading the bnd docs, you are right that this is the correct syntax.